.gitignore

```diff
@@ -1,8 +1,8 @@
-# Created by https://www.gitignore.io/api/maven,kotlin,intellij
-# Edit at https://www.gitignore.io/?templates=maven,kotlin,intellij
+# Created by https://www.toptal.com/developers/gitignore/api/maven,kotlin,intellij
+# Edit at https://www.toptal.com/developers/gitignore?templates=maven,kotlin,intellij
```

Comment:
Use the unmodified output of the command only. This file contains manual edits.

Reply:
It is almost unchanged apart - I've simply regenerated file and uncommented auto-import and idea files are driving me crazy